src/account_manager/operations/stronghold.rs

```diff
@@ -23,6 +25,14 @@ impl AccountManager {
         Ok(())
     }
 
+    /// Sets the Stronghold password clear interval
+    pub async fn set_stronghold_password_clear_interval(&self, timeout: Option<Duration>) -> crate::Result<()> {
```

Comment:
Why is this an Option ?

Reply:
If it's `None` there will be no timeout and the key will never get cleared

Reply:
Thanks